src/test/java/edu/hawaii/its/api/configuration/PropertiesFileOverrideReporterTest.java

```diff
@@ -0,0 +1,28 @@
+package edu.hawaii.its.api.configuration;
+
+import org.junit.jupiter.api.Test;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.TestPropertySource;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+@SpringBootTest(classes = { SpringBootWebApplication.class })
+@TestPropertySource(locations = "classpath:test-overrides.properties")
+public class PropertiesFileOverrideReporterTest {
+
+    @Autowired
+    private PropertiesFileOverrideReporter propertiesFileOverrideReporter;
+    private String expectedMessage;
```

Comment:
You can remove that expectedMessage class level variable, I think.

Reply:
done, thank you.